lib/ansible/module_utils/facts.py

```diff
@@ -99,6 +99,7 @@ class Facts(object):
                     ('/etc/os-release', 'SuSE'),
                     ('/etc/gentoo-release', 'Gentoo'),
                     ('/etc/os-release', 'Debian'),
+                    ('/etc/debian_version', 'Debian'),
```

Comment:
If it's not making use of the file then we shouldn't add a file here.  Instead perform the check outside of the file loop altogether.